src/axolotl/prompters.py

```diff
@@ -350,9 +350,12 @@ def _build_result(self, source):
                         "Please help us by creating an Issue to add support for this conversation type."
                     )
 
-                role = CONVERSATION_ROLE_FORMAT[self._conversation.name].format(
-                    ROLE=from_role
-                )
+                if self._conversation.name in ["llama3"]:
+                    role = from_role
```

Comment:
Why not update this instead?
```
"llama3": "{ROLE}",
```

Reply:
@teknium1 said he tried changing that 
<img width="361" alt="Screenshot 2024-07-23 at 11 10 53 PM" src="https://github.com/user-attachments/assets/2a11be06-132f-4c19-aceb-e822673fa463">